Math_Game/math_game.py

```diff
@@ -9,43 +9,37 @@
 GPIO.setmode(GPIO.BCM)
 GPIO.setup(18, GPIO.OUT)
 
-class math():
-    
+class MyMath(ABC):
     def __init__(self, a, b):
         self.a = a
         self.b = b
-    
+
     @abstractmethod
-    def operation(self):
+    def operation(self, a, b):
         pass
-    
-class add(math):
-    def __init__(self, a, b):
-        super().__init__(a, b)
-        
+
+class add(MyMath):
     def operation(self, a, b):
-        return self.a+self.b
-    
-class subtract(math):
-    def __init__(self, a, b):
-        super().__init__(a, b)
-        
+        return a+b
+
+class subtract(MyMath):
     def operation(self, a, b):
-        return self.a-self.b
+        return a-b
 
-class divide(math):
-    def __init__(self, a, b):
-        super().__init__(a, b)
-    
+class divide(MyMath):
     def operation(self, a, b):
-        return self.a/self.b
-    
-class multiply(math):
-    def __init__(self, a, b):
-        super().__init__(a, b)
-        
+        result = a/b
```

Comment:
also fixed up divide to round the result to 2 decimal points for guessing, or to a whole number if the answer is a whole number. 

* `8 / 4 = 2` user must guess 2
* `2 / 3 = 0.66` user must guess 0.66

This should actually go into an instruction prompt when running the program.

Reply:
Thanks for the comments and update Greg! 

I think it would be better to use floor division so that the answer is just a whole number to guess. What do you think?

Reply:
I think it's perfectly reasonable to keep the answer to an integer value. I think it's better to use rounding rather than floor division because personally I would have no idea what floor division meant if I saw it in the docs.

This doesn't apply to your program specifically but floor division can potentially cause bugs for negative numbers because for example the floor of 1.5 is 1 while the floor of -1.5 is 2.

Reply:
We can explicitly state in the instruction prompt that division is simply how many times can "a" divide evenly into "b". As an example, if the user selects division, a message like this would be:

For simplicity, the answer will be a whole number i.e. how many times will "a" divide evenly into "b".

Floor division would be noted in the READ.ME file.

> This doesn't apply to your program specifically but floor division can potentially cause bugs for negative numbers because for example the floor of 1.5 is 1 while the floor of -1.5 is 2.

As long as the script is generating positive numbers which is the case, there should be no issues with floor division, but this is a good tip!

Reply:
I think that's a good plan